Gaijinizer patch/Enemies.txt

```diff
@@ -1376,18 +1351,18 @@ touched once.#MTLed
 抱きつき機能自体は人形にもともと付与されていたもので
 腰振り機能発動時に自動的に相手を固定するようになっている。
 > CONTEXT: Enemies/548/BookDescription/2
-One of her milking functions is "waist swing function".
-Stimulation is sent in the crotch while feeling her soft
-skin on the whole body.
-Hug function itself was originally given to the doll
-The player is locked automatically when the waist swing
-function is activated.#MTLed
+One of her milking functions is the waist swing.
+It involves stimulating the target's crotch while
+he feels her soft skin over his whole body. Using
+the hugging function originally built into the
+doll body, she automatically keeps her opponent
+fixed in place when she activates waist swing.#Ron
```

Comment:
Minor Flow:
>_**One of her milking functions is her Waist Swing.
It involves stimulating the target's crotch while
he feels her soft skin over his whole body. Using
the hugging function originally built into the
doll body, she automatically keeps her opponent
fixed in place while she activates it.**_